### DISCUSSION TITLE
Get details of a city

### DISCUSSION THREAD

**Discussion by Mateozmaldonado17** [dr5hn/countries-states-cities-database#297] [Ideas]
Hi everyone!

I am deeply grateful for the work of the creator of this API and at the same time, I feel a duty and responsibility to improve it, even with opinions.

I feel that we should have an endpoint to be able to obtain the city and its details, even if it is the ciso and the siso of it, why?

It is important to make the information stored in the databases correct, imagine that someone daring and foolish, modifies the HTML data and sends an ID of a wrong city, incongruous that has nothing to do with the state and with the country, this will cause problems, but what if there was an endpoint where we could validate the city, let's say, we send the CISO, SISO and the ID of the city and this must give us or return a 200 with certain data, with this, we solve this problem.

Let me know if there is already a similar initiative and I will not leave without first thanking you for such a wonderful idea and collaboration, all my good energy for you.

**Comment by dr5hn** [comment]
Hi @Mateozmaldonado17 

Thanks for your suggestion.

However, This suggestion is already been raised by someone else [here](https://github.com/dr5hn/countries-states-cities-database/discussions/258).

Also, I gave a thought to this. But came to the conclusion that adding up this route will make a large number of server requests. And current server configuration is not sufficient to handle the large traffic situation.

I am still waiting on Sponsors if anyone can sponsor the Server or Help partly paying up server bills every month.

Once I get the sponsor, will surely upgrade the server and add this route.

Hope this makes sense.
